src/java.base/share/classes/com/sun/crypto/provider/CipherCore.java

```diff
@@ -446,13 +446,13 @@ AlgorithmParameters getParameters(String algName) {
         if (cipherMode == GCM_MODE) {
             algName = "GCM";
 
-            if (useNativeGCM) {
-                spec = new GCMParameterSpec
-                        (((NativeGaloisCounterMode) cipher).getTagLen()*8, iv);
+            int tagLen;
+            if (cipher instanceof NativeGaloisCounterMode) {
+                tagLen = ((NativeGaloisCounterMode) cipher).getTagLen();
             } else {
-                spec = new GCMParameterSpec
-                        (((GaloisCounterMode) cipher).getTagLen()*8, iv);
+                tagLen = ((GaloisCounterMode) cipher).getTagLen();
             }
+            spec = new GCMParameterSpec(tagLen * 8, iv);
 
```

Comment:
Please remove this blank line (456) - it isn't present upstream.

Reply:
Removed